src/aggregator/aggregator/handler/config.go

```diff
@@ -91,15 +92,17 @@ func (c FlushHandlerConfiguration) NewHandler(
 			return nil, err
 		}
 		if hc.DynamicBackend != nil {
-			sharderRouter, err := hc.DynamicBackend.NewSharderRouter(
+			handlers, sharderRouters, err = hc.DynamicBackend.newHandler(
+				handlers,
```

Comment:
I can't check sharderRouter against nil because it's a struct, unless I return pointer which is uglier, I'll keep this for now and remove the sharderRouter code once migration is done, then I can just append the handler here.